src/analysis/processing/qgsalgorithmconcavehull.cpp

```diff
@@ -50,12 +50,13 @@ QString QgsConcaveHullAlgorithm::groupId() const
 
 QString QgsConcaveHullAlgorithm::shortHelpString() const
 {
-  return QObject::tr( "This algorithm computes the concave hull of the features from an input layer." );
+  return QObject::tr( "This algorithm computes the concave hull covering all features from an input layer." ) + QStringLiteral( "\n\n" )
```

Comment:
I was about to make a PR for this so allow me to suggest it here
```suggestion
  return QObject::tr( "This algorithm computes the concave hull covering all features from an input point layer." ) + QStringLiteral( "\n\n" )
```